src/niweb/apps/noclook/templates/noclook/edit/edit_contact.html

```diff
@@ -4,6 +4,8 @@
 
 {% block js %}
 {{ block.super }}
+<link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
```

Comment:
Please download the needed css and js and put them with the static files.

Reply:
No problem, downloaded. From now on we'll keep this in mind.